8.platemap/scripts/make_platemap.r

```diff
@@ -50,7 +50,7 @@ platemap <- (
     )
     + scale_fill_manual(values = colorgrad(10))
     # change legend title
-    + labs(fill = "Stuarosporine\nDose (uM)")
+    + labs(fill = "Stuarosporine\nDose (nM)")
```

Comment:
```suggestion
    + labs(fill = "Stuarosporine\ndose (nM)")
```
Recommend lower casing dose to follow sentence case over title case.